floss/language/go/extract.py

```diff
@@ -83,7 +92,7 @@ def extract_go_strings(
                 try:
                     string = pe.get_string_at_rva(s_rva, s_size).decode("ascii")
```

Comment:
we'll have to see if we run into cases with utf-8/ascii mismatch ("strings are stored as a series of UTF8 bytes without null-terminator." - https://github.com/mandiant/flare-floss/issues/779#issuecomment-1578269595)